frontend/src/components/Psychologist/Profile/PsyProfile.jsx

```diff
@@ -33,22 +39,51 @@ const PsyProfile = () => {
       .catch(() => window.scrollTo(0, 0));
   };
 
+  const suspendPsychologist = (reason, date) => {
+    agent.Psychologist.suspend(reason, date)
+      .then(response => {
+        loadPsychologist();
+        history.push('/psychologue/mon-profil');
+        setNotification(response);
```

Comment:
When error management middleware will be merged, we would have to remove the `setNotification` right? Shouldn't we either add a TODO to remember to merge this, or merge main to this branch.

Reply:
the error management middleware was merged. And the set notification has to be specified in case of success (because some action does not require a message)